docs/rules/func-style.md

```diff
@@ -55,7 +55,7 @@ This rule has a string option:
 
 This rule has an object option for an exception:
 
-* `"allowArrowFunctions": true` (default `false`) allows the use of arrow functions (honoured only when using `declaration`)
+* When using the `"allowArrowFunctions": true` (default `false`) option with `declaration` option, arrow functions are allowed
```

Comment:
This is inconsistent with the options on line 53 and 54. Thoughts on something along these lines?
```suggestion
* `"allowArrowFunctions": true` (default `false`) When used with the `declaration` option, arrow functions are allowed
```